### DISCUSSION TITLE
Lost control of my android

### DISCUSSION THREAD

**Discussion by Kennybuck11** [community/community#53653] [New to GitHub] [answered]
### Select Topic Area

Question

### Body

I was remotely hacked how can I get control back

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Kennybuck11 , Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects so we'll be locking this post. GitHub Support can only provide support for GitHub itself.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!
